### ISSUE TITLE
Porting Entity 

### ISSUE THREAD

**Issue by docwilmot** [backdrop-ops/contrib#119]
I wanted to try RESTful Web Services module, but that is entirely tied up in Entity module, so I made a rough port of it, mostly to get the entity_metadata_wrappers function working (I actually renamed it `emw.module` with this in mind). At least that was the original idea. Now that its working, I was about to prune out the unrelated stuff, but I'm now thinking a lot of it seems like useful helper functions, like `entity_delete()` etc. 

My issue is firstly, I'd like some feedback on what to do next; and if this is fit for Backdrop contrib, or what to do to make it so please.

https://github.com/docwilmot/emw
https://github.com/docwilmot/restws

**Comment by mikemccaffrey** [comment]
Can we figure out how exactly RESTful is using the entity_metadata_wrappers, and see if there is a way to implement the same functionality more directly without the intermediary wrapper functions?

If we do need to make Entity Metadata Wrappers a contrib module, it seems more clear to call it something like entity_metadata.module, so people aren't always wondering what emw means.

**Comment by jenlampton** [comment]
> Can we figure out how exactly RESTful is using the entity_metadata_wrappers, and see if there is a way to implement the same functionality more directly without the intermediary wrapper functions?

Yes please.  I really dislike entity_metadata_wrappers and would love to get them out of backdrop :)

**Comment by argiepiano** [comment]
Side note: Entity Metadata Wrapper should not be used. Entity Plus contains this functionality, and is actively maintained. EMW is not.
